versioned_docs/version-8.4/reference/supported-environments.md

```diff
@@ -33,7 +39,36 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 
 ## Camunda 8 Self-Managed
 
-We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
```

Comment:
We recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.

Reply:
Suggest we remove highly as its ambiguous (either we make this mandatory or optional but not highly

Reply:
This is language pulled from the existing content in the Self-Managed section. I will follow up in a separate PR to change that as well.